data/text/moves.ts

```diff
@@ -2260,7 +2260,7 @@ export const MovesText: {[id: IDEntry]: MoveText} = {
 	},
 	foresight: {
 		name: "Foresight",
-		desc: "As long as the target remains active, its evasiveness stat stage is ignored during accuracy checks against it if it is greater than 0, and Normal- and Fighting-type attacks can hit the target if it is a Ghost type. Fails if the target is already affected, or affected by Miracle Eye or Odor Sleuth.",
+		desc: "As long as the target remains active, its Evasiveness stat stage is ignored during accuracy checks if it is greater than 0. Normal and Fighting type attacks can hit the target if it is a Ghost type. Fails if the target is already affected or affected by Miracle Eye or Odor Sleuth.",
```

Comment:
We don't capitalize evasiveness anywhere else.

You also shouldn't drop the hyphen. It should say Normal- or Fighting-type. It's included in [Smogon standards](https://www.smogon.com/forums/threads/spelling-and-grammar-standards.3712738/).